deepchecks/vision/metrics_utils/iou_utils.py

```diff
@@ -45,3 +48,24 @@ def compute_pairwise_ious(detected, ground_truth):
         for d_idx, d in enumerate(detected):
             ious[d_idx, g_idx] = jaccard_iou(d, g)
     return ious
+
+
+def compute_class_ious(detected, ground_truth):
```

Comment:
why not use the metric class?

Reply:
metric class aggregates on batches, and returns a response on all the data, so it doesn't make sense to use it here